ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -220,11 +220,11 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
             new BlockBody(transactions, Collections.emptyList(), maybeWithdrawals, maybeDeposits));
 
     if (maybeParentHeader.isEmpty()) {
-      LOG.atDebug()
-          .setMessage("Parent of block {} is not present, append it to backward sync")
-          .addArgument(block::toLogString)
-          .log();
-      mergeCoordinator.appendNewPayloadToSync(block);
+      //      LOG.atDebug()
+      //          .setMessage("Parent of block {} is not present, append it to backward sync")
+      //          .addArgument(block::toLogString)
+      //          .log();
```

Comment:
IMO we should still log at debug, just change the message to indicate we have dropped it

Reply:
I won't be merging this PR. If we want to progress this, we should create an issue and prioritise it.

I think we should take a more considered approach to ensure we align with the spec.
Also, there's more code we can rip out/refactor if we remove this code path.